arch/arm/src/common/arm_createstack.c

```diff
@@ -163,21 +161,31 @@ int up_create_stack(FAR struct tcb_s *tcb, size_t stack_size, uint8_t ttype)
 
   if (tcb->stack_alloc_ptr)
     {
+      uintptr_t top_of_stack;
+      size_t size_of_stack;
+
       /* The ARM uses a "full descending" stack:
        * the stack grows toward lower addresses in memory.
        * The stack pointer register points to the last pushed item in
        * the stack.
        * Items on the stack are referenced as positive word offsets from sp.
        */
 
+      top_of_stack = (uintptr_t)tcb->stack_alloc_ptr + stack_size;
+
       /* Since both stack_alloc_ptr and stack_size are in
        * STACK_ALIGNMENT, and the stack ptr is decremented before
```

Comment:
stack_alloc_ptr is not aligned anymore, is it?

Reply:
Yes, arm stack is grow toward lower address, we just need align the top of stack.

Reply:
ping @yamt

Reply:
can you update the comment?

Reply:
Done.

Reply:
ping @yamt